jre_emul/Classes/com/google/j2objc/security/IosRSASignature.java

```diff
@@ -33,6 +33,7 @@
 #include "NSDataOutputStream.h"
 #include <CommonCrypto/CommonDigest.h>
 #include <Security/Security.h>
+#import <Foundation/Foundation.h>
```

Comment:
Is this #import necessary? All j2objc headers, including NSDataOutputStream.h, should indirectly include Foundation.h, so if you found a case where it doesn't, we need to separately fix that.